maintenance/importConfig.ts

```diff
@@ -915,14 +836,15 @@ async function parseOHConfigFile(
 	assertValid(json);
 
 	const ret: Record<string, any> = {
-		_approved: json.state === "1" || json.state === 1,
+		_approved: json.approved === "1" || json.approved === 1,
+		revision: json.db_version,
 		...(json.errors?.length ? { _errors: json.errors } : undefined),
 		...(json.warnings?.length ? { _warnings: json.warnings } : undefined),
-		manufacturer: json.manufacturer,
-		manufacturerId: findManufacturerId(content, json),
+		manufacturer: json.manufacturer.label,
+		manufacturerId: formatId(json.manufacturer.reference),
 		label: sanitizeText(json.label),
 		description: sanitizeText(json.description),
-		devices: json.type_id
+		devices: json.device_ref
```

Comment:
Shouldn't this extend existing config? Does this overrides the config file?

Reply:
Yep, the OH import overwrites the original files. If someone wants to implement merging them, go ahead. I don't bother with that right now.